home-manager/neovim/default.nix

```diff
@@ -39,5 +40,6 @@ in
     vimPlugins.foreign-yazi-nvim
     vimPlugins.foreign-cyberpunk-nvim
     nvim-colorizer-lua
+    nvim-treesitter.withAllGrammars
```

Comment:
Let `nix` install parsers, otherwise trying to install ourselves results in file permission issues. 
```
Could not create parser dir ' /nix/store/.../nvim-treesitter/parser ':  Vim:E739: Cannot create directory /nix/store/.../start/nvim-treesitter/parser: permission denied 
```

[Recommendation](https://www.reddit.com/r/NixOS/comments/ytj04c/trying_to_set_up_neovim_with_treesitter_getting/) to use `.withAllGrammars`